Services/Notifications/templates/default/notifications.js

```diff
@@ -1,105 +1,86 @@
 var OSDNotifier, OSDNotifications = settings => {
-    const createThrottle = timeout => {
-        let clear = () => {};
-        return callback => {
-            clear();
-            clear = clearTimeout.bind(window, setTimeout(callback, timeout));
-        };
-    };
     const evalInCleanEnv = codeAsString => new Function('', codeAsString).call();
-    const template = (template, values) => {
-        return Object.entries(values).reduce((template, [key, value]) => {
-            return template.split('[' + key + ']').join(value);
-        }, template);
-    };
 
     const playSound = () => {
-		const sound = document.createElement('audio');
+        const sound = document.createElement('audio');
 
-		const mp3 = document.createElement('source');
-		mp3.src = 'Modules/Chatroom/sounds/receive.mp3';
-		mp3.type = 'audio/mp3';
-		sound.append(mp3);
+        const mp3 = document.createElement('source');
+        mp3.src = 'Modules/Chatroom/sounds/receive.mp3';
+        mp3.type = 'audio/mp3';
+        sound.append(mp3);
 
-		const ogg = document.createElement('source');
-		ogg.src = 'Modules/Chatroom/sounds/receive.ogg';
-		ogg.type = 'audio/ogg';
-		sound.append(ogg);
+        const ogg = document.createElement('source');
+        ogg.src = 'Modules/Chatroom/sounds/receive.ogg';
+        ogg.type = 'audio/ogg';
+        sound.append(ogg);
 
-		const attach = new Promise((resolve) => {
-			document.querySelector('body').append(sound);
-			resolve();
-		});
-		attach.then(
-			function() {
-				sound.play().then(() => {
-					console.log("Played sound successfully!");
-				}).catch((e) => {
-					console.log("Could not play sound, autoplay policy changes: https://developers.google.com/web/updates/2017/09/autoplay-policy-changes");
-					console.log(e);
-				});
-			}
-		);
+        const attach = new Promise((resolve) => {
```

Comment:
Please remove the unnecessary promise creation or explain why it is needed.

Reply:
I Agree on that but i guess more out of a conceptional way.

Initaly this promise was to ensure the sound are loaded before the sound is played. This doesnt affect "normal" setups but is recognizable with a low bandwidth.

Otherwise i assume its no wise, if a system already has a regognizable delay within that loading, to play that sound afterwards since its intention to draw the users attention to a specific time already failed then. Guess its better to miss that one out and hope for the next. 

@mjansenDatabay I would appreciate your opinion on that.

This maybe could be solve by preloading the sound file but i wouldnt recommend to preload medial content just for potential use due to high data use in general (even if thats not the case here).

However, i removed the promise.